tests/subsys/dfu/dfu_target_stream/testcase.yaml

```diff
@@ -0,0 +1,8 @@
+tests:
+  dfu.target_stream:
+    platform_allow: native_posix native_posix_64 nrf52840dk_nrf52840 nrf9160dk_nrf9160ns
```

Comment:
Curious: Why these platforms? I.e. Why both native_posix and native_posix_64, and why not more nrf platforms?

Reply:
To be frank, I have no idea. I did the native_posix_64 since I had messed up my multilib gcc on my PC, and used 64 bit to run natively. My impression is that you add the platforms needed to verify the functionality, but really I have no idea.

I will find out how this should be set and do it properly for once.

Reply:
```
platform_allow: <list of platforms>
Set of platforms that this test case should only be run for. Do not use this option to limit testing or building in CI due to time or resource constraints, this option should only be used if the test or sample can only be run on the allowed platform and nothing else.
```

So I will remove this key since this can be run on any platform.